src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/lib/create_reasonable_wait.js

```diff
@@ -1,3 +1,12 @@
-export function createReasonableWait(cb) {
-  return setTimeout(cb, 100);
+export function createReasonableWait(...promises) {
+  return Promise.all([
+    ...promises,
+    new Promise(resolve => {
+      // Make every fetch take a minimal amount of time so the user gets some feedback that something
+      // is happening.
+      setTimeout(() => {
+        resolve();
+      }, 500);
```

Comment:
It might be nice if this were configurable. Maybe change the parameters to accept:

1) An array of promises
2) A timeout (default to 300)